### PR TITLE
``gidgethub.sansio.RateLimit.from_http`` returns ``None`` if ratelimi…

### PR DESCRIPTION
**Pull request by Mariatta**
…t is not found in the headers.

Closes https://github.com/brettcannon/gidgethub/issues/25

### CODE REVIEW

File: gidgethub/sansio.py [hunk 1]
```diff
@@ -222,16 +222,19 @@ def __str__(self) -> str:
...
-        limit = int(headers["x-ratelimit-limit"])
-        remaining = int(headers["x-ratelimit-remaining"])
-        reset_epoch = float(headers["x-ratelimit-reset"])
-        return cls(limit=limit, remaining=remaining, reset_epoch=reset_epoch)
+        try:
+            limit = int(headers["x-ratelimit-limit"])
+            remaining = int(headers["x-ratelimit-remaining"])
+            reset_epoch = float(headers["x-ratelimit-reset"])
+            return cls(limit=limit, remaining=remaining, reset_epoch=reset_epoch)
+        except KeyError:
```
Comment (brettcannon) [inline gidgethub/sansio.py]: I would move this up so it was:
```python
except KeyError:
    return None
else:
    return cls(...)
```

**Comment by brettcannon** [comment]
Only a minor thing to tweak to make the code more obvious in one case.

I've also invited you to be a contributor on the repo so you don't have to wait for me to do merges on obvious stuff like your `3.7-dev` PR. 😄

**Comment by Mariatta** [comment]
Thanks for the invitation! 😄I went ahead and merged this. 🌮

**Comment by timofurrer** [comment]
Any plan to release this soon?

**Comment by brettcannon** [comment]
@timofurrer probably after #62 lands and we take for #42 since we are bumping the version number we will probably look to doing a new release.

**Pull request merged**